### PR TITLE
feat(.github): new issue templates

### PR DESCRIPTION
**Pull request by MatthewSH**
The issue templates that are in the project today are using an outdated markdown format. With the new syntax we can leverage dropdowns and more. This PR takes the old and upgrades it to the newest syntax.

### Examples
Probably worth noting that since it's a private repo, some buttons will look slightly different.

<details><summary>Screenshots</summary>
<p>

#### New Issue
image

#### Feature Request
image

#### Bug Report
image
image
</p>
</details> 

```[tasklist]
### Before Merge
- [ ] Need a better code example for bug reports
- [ ] Need to assign labels in feature request and bug report
```

### CODE REVIEW

**Comment by MatthewSH** [comment]
This is the code I use when testing new code (stripped of all my events for testing):
```javascript
import { createBot } from '@discordeno/bot'

try {
  const bot = createBot({
    token: process.env.TOKEN,
    events: {
      // ADD EVENTS NEEDED TO SHOW THE BUG HERE
    },
    // ADD INTENTS NEEDED HERE FOR YOUR TEST IF NECESSARY
    // intents: 0,
  })

  await bot.start().catch(console.error)
} catch (err) {
  console.error(err)
}
```

I added some comments back from the original bug report.

**Comment by itohatweb** [comment]
yea ok, remove the try catch though.

**Comment by Skillz4Killz** [comment]
Love it

**Comment by MatthewSH** [comment]
Updated the PR with a better code example.

File: .github/ISSUE_TEMPLATE/bug_report.yml [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+      description: Provide us, if possible, a small piece of code that's easy to test to reproduce this specific bug.
+      value: |
+        import { createBot } from '@discordeno/bot'
+
+        const bot = createBot({
+          token: process.env.TOKEN,
+          events: {
+            // ADD EVENTS NEEDED TO SHOW THE BUG HERE
+          },
+          // ADD INTENTS NEEDED HERE FOR YOUR TEST IF NECESSARY
```
Comment (itohatweb) [inline .github/ISSUE_TEMPLATE/bug_report.yml:65]:
```suggestion
            // Add the events, which are needed to show the bug
          },
          // Don't forget to add all necessary intents
```

File: .github/ISSUE_TEMPLATE/bug_report.yml [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+          // intents: 0,
```
Comment (itohatweb) [inline .github/ISSUE_TEMPLATE/bug_report.yml:66]:
```suggestion
          intents: 0,
```

**Review by itohatweb — APPROVED** [review]

**Pull request merged**